source/production/F0.Minesweeper.Logic.Abstractions/MinefieldDifficulty.cs

```diff
@@ -0,0 +1,14 @@
+namespace F0.Minesweeper.Logic.Abstractions
+{
+	public sealed record MinefieldDifficulty(uint Width, uint Height, uint MineCount)
```

Comment:
The `MinefieldGenerationOptions` also change the difficulty of the game so I suggest adding it also here. For the default difficulties I have also an idea but let's discuss in today's meeting :)

Reply:
Should we move the default difficulties to the UI, to #4?